components/common/Editor/index.js

```diff
@@ -0,0 +1,44 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import Head from 'next/head';
+import { DanteEditor } from 'Dante2/lib';
+
+import defaultContent from './content.json';
+import widgets from './widgets';
+import tooltips from './tooltips';
+import { defaultWrappers, continuousBlocks, keyCommands, characterConvertMapping } from './config';
+
+const config = {
+  el: 'dante2-editor',
+  read_only: false,
+  spellcheck: false,
+  debug: false,
+  title_placeholder: 'Title',
+  body_placeholder: 'Write your story',
+  widgets,
+  tooltips,
+  default_wrappers: defaultWrappers,
+  continuousBlocks,
```

Comment:
is it ok, that config has fields in different styles, like `continuousBlocks` and `default_wrappers`?